Resources/plugin_details.json

```diff
@@ -615,7 +615,18 @@
         "branch"          :     "master",
         "identifier"      :     "com.plexapp.plugins.mtvshows",
         "icon"            :     "mtvshows-icon.png",
-				"supporturl"			:			"https://forums.plex.tv/discussion/81430"
+        "supporturl"      :     "https://forums.plex.tv/discussion/81430"
+    },
+    {
+        "title"           :     "My Anime",
+        "bundle"          :     "MyAnime.bundle",
+        "type"            :     ["Video"],
+        "description"     :     "JapaneseMediaManager Plex plugin.",
+        "repo"            :     "https://github.com/japanesemediamanager/plexplugin",
+        "branch"          :     "master",
+        "identifier"      :     "com.plexapp.plugins.myanime",
+        "icon"            :     "icon-myanime.png",
+        "supporturl"      :     "https://forums.plex.tv/discussion/81430"
```

Comment:
This is the support URL for _MTV Shows_, not _My Anime_.
https://forums.plex.tv/discussion/91096 looks to be the original support forum.